mappings/net/minecraft/structure/ProcessorList.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5497 net/minecraft/structure/ProcessorList
```

Comment:
Even though `toString` uses `ProcessorList`, I think this class should be called `StructureProcessorList` because it holds `StructureProcessor`s.

Reply:
Nevermind, looks like we already use `PROCESSOR_LIST` in the `BuiltinRegistries` class.